packages/inventory/package.json

```
@@ -40,7 +40,7 @@
     },
     "dependencies": {
         "@redhat-cloud-services/frontend-components": "*",
-        "@redhat-cloud-services/frontend-components-notifications": "*",
+        "@redhat-cloud-services/frontend-components-notifications": "^2.0.0",
```

Can you change this to >=3.0.0
```suggestion
        "@redhat-cloud-services/frontend-components-notifications": ">=3.0.0",
```

---

@karelhala actually we can't until we migrate inventory to v3. It uses cjs imports that no longer exist. This will make it safe until we migrate.